apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.js

```diff
@@ -60,7 +61,7 @@ async function sendNotifications (context, newsItem, taskId) {
 
     checkSendingPossibility(newsItem)
 
-    const scopes = await NewsItemScope.getAll(context, { newsItem: { id: newsItem.id } })
+    const scopes = await find('NewsItemScope', { newsItem: { id: newsItem.id } })
```

Comment:
Something like this:
```suggestion
    const scopes = await find('NewsItemScope', { newsItem: { id: newsItem.id, deleledAt: null }, deleledAt: null })
```